Client/game_sa/CGameSA.cpp

```diff
@@ -33,17 +33,43 @@ float*         CGameSA::VAR_TimeStep;
 unsigned long* CGameSA::VAR_Framelimiter;
 
 unsigned int OBJECTDYNAMICINFO_MAX = *(uint32_t*)0x59FB4C != 0x90909090 ? *(uint32_t*)0x59FB4C : 160;            // default: 160
+static int m_iVehicleSunGlare;
```

Comment:
Can bool be used?

Reply:
> Can bool be used?

I've try that before, but it seems will crashed the client, unless interger

Reply:
`BOOL` is bool type with int size. Can be used here

Reply:
> `BOOL` is bool type with int size. Can be used here

yeah, done! it's works

Reply:
m_iVehicleSunGlare -> m_bVehicleSunGlare also

Reply:
> m_iVehicleSunGlare -> m_bVehicleSunGlare also

Roger that